hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/merkle/MerkleUniqueToken.java

```diff
@@ -247,7 +251,9 @@ public void setNext(final NftNumPair next) {
     }
 
     public byte[] getMetadata() {
-        return metadata;
```

Comment:
Jeszcze nie wiem jaką dokładną rolę pełnią te metadane ale dla dwóch testów były one null, wiec nie można było wywołać na nich metody `.clone()`. Imo to powinno załatwić sprawę, i też jest w miare czytelne co tu się dzieje. Niech się ktos z hedery wypowie czy tak może zostać bo jeszcze jest szansa że tamte wywalające się testy były problemem. 

Może zobaczcie sobie: `void commitsToWrappedTrackingLedgersAreRespected()` w  `AbstractLedgerWorldUpdaterTest.java`